src/modules/fstab/main.py

```diff
@@ -152,20 +152,22 @@ def generate_crypttab_line_info(self, partition):
         if not mapper_name or not luks_uuid:
             return None
 
-        if mount_point == "/":
+        if mount_point != "/":
             return None
 
         return dict(
             name=mapper_name,
             device="UUID=" + luks_uuid,
             password="/crypto_keyfile.bin",
+            options="luks,keyscript=/bin/cat",
```

Comment:
This was necessary to automatically unlock the OS filesystem without re-entering a password. All it does is use `/bin/cat` to read the contacts of `/crypto_keyfile.bin` when unlocking the filesystem. It wasn't doing this automatically with the original crypttab entry.